pallets/pool-system/src/lib.rs

```diff
@@ -835,7 +836,7 @@ pub mod pallet {
 					.expect("MaxTranches is u32. qed.");
 
 				// This kills the epoch info in storage.
-				// See: https://github.com/paritytech/substrate/blob/bea8f32e7807233ab53045fe8214427e0f136230/frame/support/src/storage/generator/map.rs#L269-L284
+				// See: https://github.com/paritytech/substrate/blob/bea8f32e7807233ab53045fe8214427e0f136230/frame/support/src/storage/generator/map.rs#L269-L284 // Broken link
```

Comment:
We should probably try to find a proper docs link for how `try_mutate` works on values with optional storage and  link that in, if we still feel we need this commented. I'd consider this "basic substrate behavior", but it's perhaps still surprising enough to a new dev that it's worth a comment, especially since we don't use this pattern often?

Reply:
I think we should remove this one. Was more me being surprised by the api ^^